process/peer/process.go

```diff
@@ -421,16 +432,38 @@ func (vs *validatorStatistics) getValidatorDataFromLeaves(
 	return validators, nil
 }
 
+func getActualList(peerAccount state.PeerAccountHandler) string {
```

Comment:
guess we can't continue without this function :D

Reply:
best function ever.